gdal/doc/gdal_virtual_file_systems.dox

```diff
@@ -668,6 +669,23 @@ Authentication options, and read-only features, are identical to
 
 @since GDAL 2.3
 
+\subsection gdal_virtual_file_systems_vsihdfs /vsihdfs/ (Hadoop File System)
+
+/vsihdfs/ is a file system handler that provides read access to HDFS.
+This handler requires that GDAL to have been built with Java support
+(--with-java) and HDFS support (--with-hdfs).
```

Comment:
You could add that only Unix build support is implemented currently

Reply:
Okay.